lib/pavilion/result/base.py

```diff
@@ -28,10 +28,12 @@ def get_sched_keys(test):
 
     return sched_keys
 
-
 BASE_RESULTS = {
     'name': lambda test: test.name,
     'id': lambda test: test.id,
+    'test_version': lambda test: test.test_version,
+    'pav_version': lambda test: test.var_man['pav.version'],
+    'compatible_pav_versions': lambda test: test.compatible_pav_versions,
```

Comment:
I don't think this one needs to be in the results. The other two do though.

Reply:
Which one? All three are highlighted.

Reply:
'compatable_pav_versions'. It looks different when you comment on multiple lines, but it's definitely confusing.